java/runtime/src/main/java/org/ray/runtime/RayNativeRuntime.java

```diff
@@ -59,15 +59,24 @@
     } catch (IOException e) {
       throw new RuntimeException("Couldn't load native libraries.", e);
     }
-    nativeSetup(RayConfig.create().logDir);
-    Runtime.getRuntime().addShutdownHook(new Thread(RayNativeRuntime::nativeShutdownHook));
-  }
 
-  public RayNativeRuntime(RayConfig rayConfig) {
-    super(rayConfig);
+    RayConfig globalRayConfig = RayConfig.create();
+    File logDirFile = new File(globalRayConfig.logDir);
+    if (!logDirFile.exists()) {
+      //noinspection ResultOfMethodCallIgnored
```

Comment:
what does this comment mean?

Reply:
It's used by IntelliJ to suppress a warning. Anyway, I've updated the code, and the comment is no longer needed.